docs/src/components/navigation/mobile-nav.tsx

```diff
@@ -119,56 +121,56 @@ export function MobileNav({ navigationMap }: NavProps) {
           </div>
 
           {/** Nav list items */}
-          <div className="px-2 pb-10">
-            <NavItem
-              level="top"
-              label="Home"
-              href="/"
-              tabIndex={navOpen ? 0 : -1}
-            />
-
-            <NavItem
-              level="top"
-              label="Docs"
-              href="/docs"
-              tabIndex={navOpen ? 0 : -1}
-              currentPage={pathname?.startsWith('/docs')}
-            />
-
-            <NavItem
-              level="top"
-              label="Blog"
-              href="/blog"
-              tabIndex={navOpen ? 0 : -1}
-              currentPage={pathname?.startsWith('/blog')}
-            />
-
-            <div className="pt-6">
-              {navigationMap?.map(({ groupName, items }) => (
-                <NavGroup key={groupName} title={groupName}>
-                  {items.map(({ label, href, title, comingSoon }) => (
-                    <NavItem
-                      key={href}
-                      label={label}
-                      href={href}
-                      title={title}
-                      level="sub"
-                      tabIndex={navOpen ? 0 : -1}
-                      comingSoon={comingSoon}
-                    />
-                  ))}
-                </NavGroup>
-              ))}
-            </div>
+          <div className="px-2" id="slideout-menu">
+            <NavGroup title="Main pages" visuallyHideTitle>
+              <NavItem
+                level="top"
+                label="Home"
+                href="/"
+                tabIndex={navOpen ? 0 : -1}
+              />
+
+              <NavItem
+                level="top"
+                label="Docs"
+                href="/docs"
+                tabIndex={navOpen ? 0 : -1}
+                currentPage={pathname === '/docs'}
```

Comment:
@jtbrolin this changes the design a little bit but also prevents the duplicate `aria-current="page"`.

i.e. we go from:
![image](https://github.com/Thinkmill/keystatic/assets/43688949/685d026c-b499-4962-8f8a-80628c4500ad)


To this:
![image](https://github.com/Thinkmill/keystatic/assets/43688949/c6575f10-e1bf-4be3-af93-4bff1c24965c)


Let me know if that's ok (to me it makes more sense but is inconsistent with desktop view).

Reply:
Yeah OK with me (I didn't notice the duplicate `aria-current="page"`, was too focused on the desktop<->mobile parity 😅)

Reply:
Lol all good! I didn't notice until a follow-up look at something else this morning!